src/general/OptionsMenu.cs

```diff
@@ -258,32 +273,87 @@ public override void _Ready()
         miscTab = GetNode<Control>(MiscTabPath);
         playIntro = GetNode<CheckBox>(PlayIntroPath);
         playMicrobeIntro = GetNode<CheckBox>(PlayMicrobeIntroPath);
-        tutorialsEnabled = GetNode<CheckBox>(TutorialsEnabledPath);
+        tutorialsEnabledOnNewGame = GetNode<CheckBox>(TutorialsEnabledOnNewGamePath);
         cheats = GetNode<CheckBox>(CheatsPath);
         autosave = GetNode<CheckBox>(AutoSavePath);
         maxAutosaves = GetNode<SpinBox>(MaxAutoSavesPath);
         maxQuicksaves = GetNode<SpinBox>(MaxQuickSavesPath);
+        localSettingsVBox = GetNode<VBoxContainer>(LocalSettingsVBoxPath);
+        tutorialsEnabled = GetNode<CheckBox>(TutorialsEnabledPath);
 
         backConfirmationBox = GetNode<WindowDialog>(BackConfirmationBoxPath);
         defaultsConfirmationBox = GetNode<ConfirmationDialog>(DefaultsConfirmationBoxPath);
         errorAcceptBox = GetNode<AcceptDialog>(ErrorAcceptBoxPath);
 
         selectedOptionsTab = SelectedOptionsTab.Graphics;
+    }
+
+    public override void _Process(float delta)
```

Comment:
Empty method

Reply:
It's like that in master, am I supposed to remove it? I think it's only showing as a diff because it got moved.

Reply:
Please remove it, it unnecessarily runs each frame.